File: 프로그래머스/0/120887. k의 개수/k의 개수.cs
using System;
using System.Linq;
public class Solution 
{
    public int solution(int i, int j, int k) 
    {
        string strNum = "";
        int count = 0;
        for(int num=i; num<=j; num++) {
            strNum = num.ToString();
            for(int ln=0; ln<strNum.Length; ln++) {
                if(strNum[ln].ToString() == k.ToString()) {
                    count++;
                }    
            }
            strNum = "";
        }
        return count;
    }
}

File: 프로그래머스/0/120887. k의 개수/k의 개수.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(int i, int j, int k) 
{
    string num = "";
    for(int idx = i; idx<=j; idx++)
    {
        num += to_string(idx);
    }
    
    int count = 0;
    for(char &c : num)
    {
        if(c-'0' == k)
        {
            count++;
        }
    }
    return count;
}